mobly/test_runner.py

```diff
@@ -75,12 +76,13 @@ def main(argv=None):
       runner.add_test_class(config, test_class, tests)
       try:
         runner.run()
-        ok = runner.results.is_all_pass and ok
```

Comment:
why remove this one?

Reply:
Please reply comments in their original thread.

It sounds like in that case the test should pass, so it's expected behavior, no?

In any case, that's a different change; should be a separate PR if needed

Reply:
Reverted.